test/unit/mock.c

```diff
@@ -1,3 +1,3 @@
 #include "unity.h"
-void setUp(void) {}
-void tearDown(void) {}
\ No newline at end of file
+// void setUp(void) {}
```

Comment:
I am not sure. It seems there is something wrong with the cmake configuration.
I am using these functions in the new `network_channels_test.c` and the compiler was complaining about multiple implementations for these functions.

Reply:
By commenting out these implementations in mock.c it compiled.

Reply:
Hm, on my end I get undefined reference errors when these functions are not there...

Reply:
Ah, I just defined them that way because we are not using them for any other test